images/busybox/alpine.tf

```diff
@@ -0,0 +1,32 @@
+module "latest-alpine" {
+  providers = {
+    apko = apko.alpine
+  }
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.alpine.apko.yaml")
+  extra_packages    = []
```

Comment:
Ah, this must be the key!  we default this to `wolfi-baselayout` 🤦 .  That's subtle, can you add a comment?